.travis.yml

```diff
@@ -203,7 +203,8 @@ script:
   - opt_enable_adv Z_DUAL_STEPPER_DRIVERS Z_DUAL_ENDSTOPS BEZIER_CURVE_SUPPORT EXPERIMENTAL_I2CBUS
   - opt_set_adv I2C_SLAVE_ADDRESS 63
   - opt_enable_adv ADVANCED_PAUSE_FEATURE PARK_HEAD_ON_PAUSE LCD_INFO_MENU
-  - opt_add_adv Z2_MIN_PIN 2
+  - pins_set RAMPS X_MAX_PIN -1
+  - opt_add_adv Z2_MAX_PIN 2
```

Comment:
What's the issue here?

Reply:
I suppose you noticed that this is not building for `MINIRAMBO`, nor does it enable `PWM_MOTOR_CURRENT`.

Reply:
The way it was, it would put 500 lines of warnings into the travis log because it was defining Z2_MIN_PIN multiple times.  I will make the comment and the build reflect each other.

Reply:
At the time when I added `opt_add_adv Z2_MIN_PIN 2` the compiler error was that no `Z2_MIN_PIN` was defined. It would be a good idea to go to the Blame log and find out when changes were made to these lines and what caused the new errors to start showing up.